packages/babel-helpers/src/helpers/usingCtx.ts

```diff
@@ -0,0 +1,81 @@
+/* @minVersion 7.23.0 */
+
+type Stack = {
+  v: any;
+  d: () => any;
+  a: boolean;
+};
+
+export default function _usingCtx() {
+  var _disposeSuppressedError =
+      typeof SuppressedError === "function"
+        ? // eslint-disable-next-line no-undef
+          SuppressedError
+        : (function (suppressed: boolean, error: Error) {
+            var err = new Error() as SuppressedError;
+            err.name = "SuppressedError";
+            err.suppressed = suppressed;
+            err.error = error;
+            return err;
+          } as SuppressedErrorConstructor),
```

Comment:
I borrowed from ts's implementation because theirs is much simpler, I think the behavior has some differences in the prototype, but I'm not sure who is right and if it is acceptable.

Also sorry for making these changes after the review. :) @JLHwung 

ts helper
```js
var __disposeResources = (this && this.__disposeResources) || (function (SuppressedError) {
    return function (env) {
        function fail(e) {
            env.error = env.hasError ? new SuppressedError(e, env.error, "An error was suppressed during disposal.") : e;
            env.hasError = true;
        }
        function next() {
            while (env.stack.length) {
                var rec = env.stack.pop();
                try {
                    var result = rec.dispose && rec.dispose.call(rec.value);
                    if (rec.async)
                        return Promise.resolve(result).then(next, function (e) {
                            fail(e);
                            return next();
                        });
                } catch (e) {
                    fail(e);
                }
            }
            if (env.hasError)
                throw env.error;
        }
        return next();
    };
})(typeof SuppressedError === "function" ? SuppressedError : function (error, suppressed, message) {
    var e = new Error(message);
    return e.name = "SuppressedError",
    e.error = error,
    e.suppressed = suppressed,
    e;
});
```

Reply:
https://github.com/es-shims/SuppressedError/blob/11f592cc4ec88691b20944057b1772e4b62d201c/implementation.js#L13

By looking at it, I think the previous implementation was more spec-compliant, but it seems like ts's behavior is sufficient for use here.
Since we did not assign a value to `SuppressedError`, users should not use `instanceof` to compare it.